### DISCUSSION TITLE
Kibana Reporting Issues

### DISCUSSION THREAD

**Discussion by DoYouKnowWhoElse** [telekom-security/tpotce#1711] [Q&A]
ReportingError(code: browser_could_not_launch_error)

How could I fix this? I need to export a PNG and PDF of the graphs.

**Comment by t3chn0m4g3** [comment]
Both are [subscription features](https://www.elastic.co/guide/en/kibana/current/reporting-getting-started.html) and thus unavailable in T-Pot due to license restrictions.

**Reply by DoYouKnowWhoElse** [reply]
I started the free trial though which gives access to them. If its still an issue, any other way I could potentially export the attack logs then for two weeks worth of data.

**Reply by DoYouKnowWhoElse** [reply]
@t3chn0m4g3 Would you know how to export/extract attack logs into a single file for every honeypot?

**Reply by t3chn0m4g3** [reply]
You can follow the example in [logstash.conf](https://github.com/telekom-security/tpotce/blob/6136cf320608fbcd8e9191450493719ca0b9480c/docker/elk/logstash/dist/logstash.conf#L717-L744).
